.bandit

```diff
@@ -1 +1 @@
-skips: ['B101']
+skips: ['B101', 'B322']
```

Comment:
B322 Refers to the method `input` which the bandit message asserts is safe in Python3, but complains about anyway. https://github.com/PyCQA/bandit/issues/402

Reply:
Fine.